source/renderer/app/components/staking/stake-pools/StakePoolsTable.scss

```diff
@@ -105,7 +105,7 @@
       padding: 0 20px;
       position: sticky;
       top: 0;
-      z-index: 999;
+      z-index: 1;
```

Comment:
Looks good! FYI: i created this file to have a better control over zIndex configuration:
source/renderer/app/themes/mixins/layers.scss

Reply:
Lets use that!